src/test/java/org/testeditor/fixture/host/s3279/StatusTest.java

```diff
@@ -0,0 +1,199 @@
+                                                                                               // terminal
+                                                                                               // mode
+                                                                                               // 8
+                                                                                               // is
+                                                                                               // illegal
+        new Status(statusString);
+    }
+
+    @Test
+    public void numberOfRowsSuccessfulTest() {
+        String statusString = defaultStatusString;
+        Status status = new Status(statusString);
+        Assert.assertTrue(status.getNumberRows() == 24);
+    }
+
+    @Test
+    public void numberOfRowsUnSuccessfulTest() {
+        String statusString = defaultStatusString;
+        Status status = new Status(statusString);
+        Assert.assertFalse(status.getNumberRows() == 32);
+    }
+
+    @Test
+    public void numberOfColumnsSuccessfulTest() {
+        String statusString = defaultStatusString;
+        Status status = new Status(statusString);
+        Assert.assertTrue(status.getNumberColumns() == 80);
+    }
+
+    @Test
+    public void numberOfColumnsUnSuccessfulTest() {
+        String statusString = defaultStatusString;
+        Status status = new Status(statusString);
+        Assert.assertFalse(status.getNumberColumns() == 81);
+    }
+
+    @Test
+    public void currentColumnSuccessfulTest() {
+        String statusString = defaultStatusString;
+        Status status = new Status(statusString);
+        Assert.assertTrue(status.getCurrentCursorColumn() == 44);
+    }
+
+    @Test
+    public void currentColumnUnSuccessfulTest() {
+        String statusString = defaultStatusString;
+        Status status = new Status(statusString);
+        Assert.assertFalse(status.getCurrentCursorColumn() == 45);
+    }
+
+    @Test
+    public void currentRowSuccessfulTest() {
+        String statusString = defaultStatusString;
+        Status status = new Status(statusString);
+        Assert.assertTrue(status.getCurrentCursorRow() == 6);
+    }
+
+    @Test
+    public void currentRowUnSuccessfulTest() {
+        String statusString = defaultStatusString;
+        Status status = new Status(statusString);
+        Assert.assertFalse(status.getCurrentCursorRow() == 7);
+    }
+
+    @Test
+    public void windowIdSuccessfulTest() {
+        String statusString = defaultStatusString;
+        Status status = new Status(statusString);
+        Assert.assertTrue(status.getWindowId().equals("0x0"));
+    }
+
+    @Test
+    public void windowIdUnSuccessfulTest() {
+        String statusString = defaultStatusString;
+        Status status = new Status(statusString);
+        Assert.assertFalse(status.getWindowId().equals("WATT"));
```

Comment:
testing equals method here (cannot be equal to "0x0" and "WATT" at the same time)